src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -208,6 +300,76 @@
   }
 }
 
+// Task list item
+.pf-c-notification-drawer__task-list-item {
+  position: relative;
+  display: flex;
+  align-items: center;
+  min-height: var(--pf-c-notification-drawer__task-list-item--MinHeight);
+  padding: var(--pf-c-notification-drawer__task-list-item--PaddingTop) var(--pf-c-notification-drawer__task-list-item--PaddingRight) var(--pf-c-notification-drawer__task-list-item--PaddingBottom) var(--pf-c-notification-drawer__task-list-item--PaddingLeft);
+  background-color: var(--pf-c-notification-drawer__task-list-item--BackgroundColor);
+  outline-offset: var(--pf-c-notification-drawer__task-list-item--OutlineOffset);
+
+  &.pf-m-info {
+    --pf-c-notification-drawer__task-list-item-status--Color: var(--pf-c-notification-drawer__task-list-item--m-info__task-list-item-status--Color);
+  }
+
+  &.pf-m-warning {
+    --pf-c-notification-drawer__task-list-item-status--Color: var(--pf-c-notification-drawer__task-list-item--m-warning__task-list-item-status--Color);
+  }
+
+  &.pf-m-danger {
+    --pf-c-notification-drawer__task-list-item-status--Color: var(--pf-c-notification-drawer__task-list-item--m-danger__task-list-item-status--Color);
+  }
+
+  &.pf-m-success {
+    --pf-c-notification-drawer__task-list-item-status--Color: var(--pf-c-notification-drawer__task-list-item--m-success__task-list-item-status--Color);
+  }
+
+  &.pf-m-default {
+    --pf-c-notification-drawer__task-list-item-status--Color: var(--pf-c-notification-drawer__task-list-item--m-default__task-list-item-status--Color);
+  }
+
+  &.pf-m-hoverable {
+    cursor: pointer;
+
+    &:hover {
+      z-index: var(--pf-c-notification-drawer__task-list-item--m-hoverable--hover--ZIndex);
+      box-shadow: var(--pf-c-notification-drawer__task-list-item--m-hoverable--hover--BoxShadow);
+    }
+
+    &:hover,
+    &:focus,
+    &:active {
+      --pf-c-notification-drawer__task-list-item-hover-action--Display: inline-block;
+    }
+  }
+}
+
+// Task list item pieces
+.pf-c-notification-drawer__task-list-item-title {
+  margin-right: var(--pf-c-notification-drawer__task-list-item-title--MarginRight);
+}
+
+.pf-c-notification-drawer__task-list-item-hover-action {
+  display: var(--pf-c-notification-drawer__task-list-item-hover-action--Display);
+  margin-right: var(--pf-c-notification-drawer__task-list-item-hover-action--MarginRight);
+}
+
+.pf-c-notification-drawer__task-list-item-status {
+  margin-left: auto;
+  color: var(--pf-c-notification-drawer__task-list-item-status--Color);
+  text-align: end;
```

Comment:
Is this necessary if it's always pushed to the right? Also it's totally fine if you want to use a logical property, but we don't really use them anywhere as far as I'm aware, except for flex/grid position.